src/routes/JobSummary/components/cells/HoldingsCell.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  getRecordActionStatusLabel,
+  getHotlinkCellFormatter,
+  BaseLineCell,
+} from '../utils';
+
+import { RECORD_ACTION_STATUS } from '../../../../utils';
+
+export const HoldingsCell = ({
+  instanceId,
+  holdingsInfo,
+  itemInfo,
+  locations,
+}) => {
+  const holdingsInfoCell = holdingsInfo?.map((holdings, index) => {
+    const entityLabel = getRecordActionStatusLabel(holdings.actionStatus);
+    const holdingsId = holdings.id;
+    const path = `/inventory/view/${instanceId}/${holdingsId}`;
+    const locationId = holdings.permanentLocationId;
+    const isDiscarded = holdings.actionStatus === 'DISCARDED';
```

Comment:
@OleksandrHladchenko1 
```suggestion
    const isDiscarded = holdings.actionStatus === RECORD_ACTION_STATUS.DISCARDED;
```